assets/js/signup.js

```diff
@@ -1,101 +1,92 @@
 $(document).ready(function() {
-  // loadQuestions();
+	// loadQuestions();
 
-  // to be deleted at a later point
-  validateSignupForm();
+	// to be deleted at a later point
+	validateSignupForm();
 
-  // Since non-required fields are "valid" when they are empty, we need an
-  // alternate way to keep labels raised when there is content in their
-  // associated input field
-  $(".vita-form-textfield input").blur(function() {
-    var isBlank = $.trim($(this).val()).length > 0;
-    $label = $(this).siblings(".vita-form-label").toggleClass( "vita-form-label__floating", isBlank );
-  });
+	// Since non-required fields are "valid" when they are empty, we need an
+	// alternate way to keep labels raised when there is content in their
+	// associated input field
+	$(".vita-form-textfield input").blur(function() {
+		var isBlank = $.trim($(this).val()).length > 0;
+		$label = $(this).siblings(".vita-form-label").toggleClass( "vita-form-label__floating", isBlank );
+	});
 
 });
 
 function validateSignupForm() {
-  $("#vitaSignupForm").validate({
-    rules: {
-      "firstName": "required",
-      "lastName": "required",
-      email: {
-        required: true,
-        email: true
-      },
-      phone: {
-        required: true,
-        phoneUS: true
-      }
-    },
-    messages: {
-      email: {
-        required: "We need your email address to confirm your appointment",
-        email: "Your email address must be in the format of name@domain.com"
-      }
-    }
-  });
+	$("#vitaSignupForm").validate({
+		rules: {
+			"firstName": "required",
+			"lastName": "required",
+			email: {
+				required: true,
+				email: true
+			},
+			phone: {
+				required: true,
+				// phoneUS: true
```

Comment:
Forgot to comment on this.

You have to include a separate lib to use jquery phone validation(for some reason)
https://stackoverflow.com/questions/16652776/jquery-validation-plugin-phoneus-field-is-breaking-form#answer-16652933

And in general I try to avoid over-validating(as I see it) input that doesn't really need validation. Maybe make sure there are at least 10 numbers but I feel that anything beyond that degrades UX needlessly.